src/js/utils/fn.js

```diff
@@ -47,13 +51,14 @@ export const bind = function(context, fn, uid) {
  * Wraps the given function, `fn`, with a new function that only invokes `fn`
  * at most once per every `wait` milliseconds.
  *
- * @param  {Function} fn
- *         The function to be throttled.
+ * @function
```

Comment:
In cases where functions are anonymously assigned to variables, `@function` is needed.